src/App.js

```diff
@@ -1,20 +1,67 @@
-import React from "react";
-import logo from "./logo.png";
+import React, { useState } from "react";
 import "./App.css";
+import axios from "axios";
 
-class App extends React.Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-        </header>
+function App() {
+  const [input, setInput] = useState("");
+  const [data, setData] = useState({});
+  const [cityName, setCityName] = useState("");
+  const API_KEY = "7c52b0b346e3515a9d1950f5fd508f09";
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    //call axios
+    axios
+      .get(
+        `https://api.openweathermap.org/geo/1.0/direct?q=${input}&limit=1&appid=${API_KEY}`
+      )
+      .then((response) => response.data[0])
+      .then((geoData) => {
+        setCityName(geoData.name);
+        axios
+          .get(
+            `https://api.openweathermap.org/data/2.5/forecast?lat=${geoData.lat}&lon=${geoData.lon}&appid=${API_KEY}`
+          )
+          .then((response) => {
+            setData(response.data.list[0].main);
```

Comment:
I don't think this works, as you do not return the result of the axios request in the previous `.then` block